### ISSUE TITLE
Want to use custom schema name for created database tables

### ISSUE THREAD

**Issue by kaysweebe** [farahats9/sqlalchemy-celery-beat#17]
TLDR; Using custom schema name does not allow me to insert without getting operational errors because the insert tries to use the default schema name given on the model object. 

Example: CrontabSchedule.__table_args__.schema = 'celery_schema'

Currently, the only schema name supported is `celery_schema`. I can add these tables super efficiently into my database with schema `my_example_schema` so that in my database it shows `my_example_schema.celery_crontabschedule` using the normal start up process; however, the `celery_schema` name is hard-coded on each of the models making it really difficult to use a custom schema name.

**Comment by kaysweebe** [comment]
I actually have a fix for this now. Not too sure if you allow others to open PR's on this repo?

**Comment by farahats9** [comment]
Any PR is welcome for sure.

But did you try to use the `beat_schema` config variable in the celery beat config dict?
it would looks something like this:
```python
config = {
    'beat_schedule': {
        "run_every_hour": {
            "task": "tasks.run_every_hour",
            "schedule": schedules.crontab(hour="*", minute=0),
        },
    },
    'beat_dburi': 'mysql+mysqldb://user:password@127.0.0.1:3306/test',
    'beat_schema': 'my_example_schema',
    ...
}

celery.conf.update(config)
```
I don't remember if it works well on MySQL or if the database ignores it, but the first run of celery beat it should create the tables in the schema you described. 
If you have a better solution please go ahead and open a PR.

**Comment by farahats9** [comment]
Closing this for lack of response.
